Gemfile.lock

```diff
@@ -495,9 +495,8 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    websocket (1.2.4)
-    xpath (2.0.0)
-      nokogiri (~> 1.3)
+    xpath (3.2.0)
+      nokogiri (~> 1.8)
```

Comment:
I did have an issue and I thought that if something was not locked to a version it would not be a concern.  Would you like me to roll this back?

Reply:
Okay! What issue did you have? If it was with a particular gem maybe we can isolate the issue?

In theory specifying versions in Gemfile would prevent issues but in practice we need to be careful about doing a bundle update, especially those major version changes. It's possible that the upgrades are totally fine, but ideally we should do due diligence and check the changelogs of the gems that are changing to make sure. Since that effort isn't really scheduled as part of your task I think it makes sense to roll it back.

Reply:
In the end I think it was a missing library needed for ruby which means I did not have to change the versions.  I will Put them back to the way they were.